refract-spec.md

```diff
@@ -54,13 +56,13 @@ The Refract Element contains three properties, `element`, `attributes`, and `con
 
 #### Example
 
-An simple element may look like this, where `foo` is the element type, `id` is an attribute for the `foo` element, and `content` is a string with a value of `bar`.
+An element may look like this, where `foo` is the element type, `id` is an attribute for the `foo` element, and `content` is a string with a value of `bar`. Here, the `id` is "baz" and may be used for referencing.
 
 ```json
 {
   "element": "foo",
   "attributes": {
-    "id": 4
```

Comment:
@smizell The problem this example does not represent this JSON example: https://github.com/refractproject/refract-spec/blob/smizell/referencing-namespacing-extending/refract-spec.md#json-representation but the other example is correct.